pkg/maintenance/alertmanagerMaintenance.go

```diff
@@ -157,3 +186,14 @@ func createDefaultMatchers() []*amv2Models.Matcher {
 	inNamespaceAlertMatcher := createMatcher("namespace", "(^openshift.*|^kube.*|^redhat.*|^default$)", true)
 	return amv2Models.Matchers{nonCriticalAlertMatcher, inNamespaceAlertMatcher}
 }
+
+// Check if a silence with comment exists
+func silenceExists(mList *amSilence.GetSilencesOK, comment string) bool {
```

Comment:
If an expired silence exists will it still return true here? does it also need to check payload.Status? 
(haven't confirmed if this is an issue, just wondering)

Reply:
It does return expired silences. Each silence should only be created once and If it expires we get alerted for it.
If we check for active we can end up recreating a maintenance.